cedar-wasm/README.md

```diff
@@ -28,7 +28,7 @@ console.log(cedar.getCedarVersion());
 Importing the esm version using esm async import:
 
 ```
-import('@cedar-policy/cedar-wasm')
+import('@cedar-policy/cedar-wasm/nodejs')
```

Comment:
Are you sure this is needed after the other change? Did you test?

Reply:
@Swolebrain Yes, without `/nodejs` you get the following error:
```
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".wasm" for /node_modules/@cedar-policy/cedar-wasm/esm/cedar_wasm_bg.wasm
```